plugin/telegram.py

```diff
@@ -64,3 +64,21 @@ def pocsag(self, bwPacket):
                 logging.exception("Error while sending Telegram Message")
             except Exception as e:
                 logging.exception("Unknown Error while sending Telegram Message: " + str(type(e).__name__) + ": " + str(e))
+
+    def zvei(self, bwPacket):
+        """!Called on ZVEI alarm
+        @param bwPacket: bwPacket instance"""
+        msg = self.parseWildcards(self.config.get("message"))
+        for chatId in self.config.get("chatIds", default=[]):
+            try:
```

Comment:
Das senden der reinen Nachricht ist ja bei allen Typen gleich, sollte man also in ne Funktion auslagern.
Unterschied ist ja nur, dass Pocsag ggf auch eine Position senden kann, das müsste man halt separat machen